spec/customer_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'customer'
+
+describe Customer do
+
+    it 'can see a list of dishes and prices' do
+      expect(subject.see_menu).to eq({ 'plantain fries' => 3,
+                                        'banana curry'=> 8,
+                                        'banoffee pie'=> 5,
+                                        'banana milkshake'=> 3})
+  end
```

Comment:
Lint/BlockAlignment: `end` at 10, 2 is not aligned with `it 'can see a list of dishes and prices' do` at 5, 4